src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,24 @@
+package calculator
+
+class StringAddCalculator {
+
+    fun run(input: String): Int {
+        try {
+            return calculate(input)
+        } catch (e: RuntimeException) {
+            throw RuntimeException(e.message, e)
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return calculate(input)
```